modules/express/src/config.ts

```diff
@@ -102,7 +105,7 @@ export const DefaultConfig: Config = {
  * @param url
  * @return {string}
  */
-function _forceSecureUrl(url: string): string {
+export function _forceSecureUrl(url: string): string {
```

Comment:
the leading `_` usually means "private"
```suggestion
export function forceSecureUrl(url: string): string {
```